android/modules/ui/src/java/ti/modules/titanium/ui/widget/tabgroup/TiUIBottomNavigationTabGroup.java

```diff
@@ -256,10 +248,12 @@ public void removeTabItemFromController(int position)
 	public void selectTabItemInController(int position)
 	{
 		// Fire the UNSELECTED event from the currently selected tab.
-		((TabGroupProxy) getProxy())
-			.getTabList()
-			.get(currentlySelectedIndex)
-			.fireEvent(TiC.EVENT_UNSELECTED, null, false);
+		if (currentlySelectedIndex != -1) {
+			((TabGroupProxy) getProxy())
+				.getTabList()
+				.get(currentlySelectedIndex)
+				.fireEvent(TiC.EVENT_UNSELECTED, null, false);
+		}
```

Comment:
We should check if `getProxy()` returns `null` in the `selectTabItemInController()` method and the `onMenuItemClick()` method, which can happen when the proxy has been released (ie: it's JavaScript object was garbage collected).

Reply:
Updated.